tools/coverage/docs/runLocally.md

```diff
@@ -0,0 +1,15 @@
+A pre-submit check, to see how the coverage of your local repository compare to the lastest postsubmit run, can be run locally in the following way
+
+1. download https://github.com/kubernetes/test-infra/tree/master/robots/coverage
```

Comment:
The first two steps can probably be replaced by:

`go get github.com/kubernetes/test-infra/robots/coverage`